daemon/daemon.go

```diff
@@ -1389,6 +1389,7 @@ func (d *Daemon) bootstrapFQDN(restoredEndpoints *endpointRestoreState) (err err
 	if option.Config.ToFQDNsEnablePoller {
 		fqdn.StartDNSPoller(d.dnsPoller)
 	}
+	dnsproxy.SetRejectReply(option.Config.FQDNRejectResponse)
```

Comment:
Since this is a configuration thing, could you put it just before we init the proxy? Alternatively, it can go with cfg above.